editor/src/clj/editor/model_scene.clj

```diff
@@ -119,47 +120,56 @@
 
     out-indices))
 
+(defn- flat->vectors [^long component-count interleaved-component-values]
+  (let [vectors (into []
+                      (coll/partition-all-primitives :double component-count)
+                      interleaved-component-values)]
+    ;; Return nil if the interleaved component values cannot be evenly
+    ;; partitioned into the desired component count.
+    (when (or (coll/empty? vectors)
+              (= component-count (count (peek vectors))))
+      vectors)))
+
 (defn mesh->renderable-data [mesh]
-  (let [positions (doubles->floats (:positions mesh))
-        normals (doubles->floats (:normals mesh))
-        texcoord0s (doubles->floats (:texcoord0 mesh))
+  (let [positions (flat->vectors 3 (:positions mesh))
+        normals (flat->vectors 3 (:normals mesh))
+        tangents (flat->vectors 3 (:tangents mesh))
```

Comment:
Apparently there are only three components to the `tangents` we import from the meshes, and there are no `binormals`. So how are you supposed to reconstruct the tangent space? 🤔

Either we need the `binormals`, or we could have a handedness scalar [`-1.0` or `1.0`] in `tangent.w` to inverse it for purposes of the cross-product with the `normal`.

For now, I'll just pass it on to the shader, I guess... 🤷

Reply:
Yeah, it's a good question. Afaik blender exports handedness in the w component, which would be the simplest fix for it. I wonder if this test shows handedness issues: https://github.com/KhronosGroup/glTF-Sample-Models/blob/main/2.0/NormalTangentTest/README.md

In any case, we don't have to fix it now but I'll add an issue for it